ymcatwincities/openy
# CSS Fix for subcategory images in mobiles.

Another fix for: https://github.com/ymcatwincities/openy/issues/2539

Steps to test:
- Visit an openy site.
- Go to the program subcategory page preferably with category listing block present in that.
- Now, adjust the content for the program subcategories that displayed in that page and reduce the description to two or three words and save the content. Next, refresh the page.
- Adjust the browser to different breakpoints for mobile devices to check the fix for the issue

**Note:** Test this fix against carnation theme for now. I am working on the fix for rose/lily theme.
